packages/cli/src/InternalHooks.ts

```diff
@@ -834,31 +833,4 @@ export class InternalHooks {
 	}): Promise<void> {
 		return await this.telemetry.track('User updated external secrets settings', saveData);
 	}
-
-	async onTeamProjectCreated(data: { user_id: string; role: GlobalRole }) {
-		return await this.telemetry.track('User created project', data);
-	}
-
-	async onTeamProjectDeleted(data: {
-		user_id: string;
-		role: GlobalRole;
-		project_id: string;
-		removal_type: 'delete' | 'transfer';
-		target_project_id?: string;
-	}) {
-		return await this.telemetry.track('User deleted project', data);
-	}
-
-	async onTeamProjectUpdated(data: {
-		user_id: string;
-		role: GlobalRole;
-		project_id: string;
-		members: Array<{ user_id: string; role: ProjectRole }>;
-	}) {
-		return await this.telemetry.track('Project settings updated', data);
-	}
-
-	async onConcurrencyLimitHit({ threshold }: { threshold: number }) {
-		await this.telemetry.track('User hit concurrency limit', { threshold });
-	}
```

Comment:
This hook was unused. The telemetry service method is called directly in `ConcurrencyControlService` because adding `InternalHooks` to it caused a dep cycle.